pkg/asset/installconfig/platformcredscheck.go

```diff
@@ -68,6 +68,12 @@ func (a *PlatformCredsCheck) Generate(dependencies asset.Parents) error {
 		}
 	case ibmcloud.Name:
 		_, err = ibmcloudconfig.NewClient()
+	case powervs.Name:
```

Comment:
This appears to have removed the error handling from the case `ibmcloud.Name`.

Reply:
Oh.. it's a mistake, I will add it back

Reply:
updated